CHANGES.txt

```diff
@@ -33,6 +33,10 @@ Changes
 
  - Changed the Enterprise License missing check message, as well as lowering
    the severity from HIGH to LOW.
+   
+  - Introduce support for single row subselects in ANY, e.g.::
+
+      Select id from t where id = ANY(Select id from t2)
```

Comment:
It is a single row-single column subselect. Will rephrase it to `single-column`.